server/middlewares/validatePostRecords.js

```diff
@@ -14,19 +13,19 @@ export const validateArrayValues = (req, res, next) => {
 	const urlArray = [images, videos];
 
 	if (Helpers.isNotArray(urlArray)) {
-		Helpers.returnForError(req, res, 400, "invalid input");
+		Helpers.returnForError(req, res, 400, "images or videos not an array");
 
 	} else if (Helpers.isStringInsideArray(images)) {
-		Helpers.returnForError(req, res, 400, "invalid input");
+		Helpers.returnForError(req, res, 400, "images content is not a string");
 
 	} else if (Helpers.isStringInsideArray(videos)) {
-		Helpers.returnForError(req, res, 400, "invalid input");
+		Helpers.returnForError(req, res, 400, "videos content is not a string");
 
 	} else if (Helpers.isValueInsideArrayEmpty(images)) {
-		Helpers.returnForError(req, res, 400, "undefined input");
+		Helpers.returnForError(req, res, 400, "undefined input images");
```

Comment:
I also don't think your response messages should contain the word `undefined`. Try to make them as non-technical as possible.